tests/helpers/test_package_util.py

```diff
@@ -0,0 +1,23 @@
+from ..commands.run.util import get_command, create_ins
+from ..fixtures import *
+from sinol_make.helpers import package_util
+
+
+def test_get_task_id(create_package):
+    assert package_util.get_task_id() == "abc"
+
+
+def test_extract_test_no():
+    assert package_util.extract_test_no("in/abc1a.in") == "1a"
```

Comment:
Yes, otherwise this directory wouldn't be considered a module and files from it couldn't be imported. More info: https://docs.python.org/3/reference/import.html#regular-packages

Reply:
It will return `12ca`. I added some comments for this function 29b1859ab287eddfed44a57070adaa0e0a6cc597 and tests b6608de9a788d91adc33b781fa07ce78a96936e5 that should make this clearer.